### PR TITLE
Fixup of ci_job namespace

### PR DESCRIPTION
**Pull request by t0ffel**

### CODE REVIEW

File: templates/ci_model/com.redhat.viaq-ci.template.json [hunk 1]
```diff
@@ -98,6 +93,11 @@
               "index": "not_analyzed",
               "type": "string"
             },
+            "number": {
+              "doc_values": true,
+              "index": "not_analyzed",
```
Comment (portante) [inline templates/ci_model/com.redhat.viaq-ci.template.json:98]: Do we need this for a string type?

**Review by portante — COMMENTED** [review]
Seems fine, just need to consider that `not_analyzed` attribute on an `integer` data type.

**Comment by richm** [comment]
@t0ffel any update on this PR?

File: templates/ci_model/com.redhat.viaq-ci.template.json (hunk 1 above)
Comment (lukas-vlcek) [reply templates/ci_model/com.redhat.viaq-ci.template.json:98]: @portante Did you mean to ask "Do we need this for an *integer* type?"
<https://www.elastic.co/guide/en/elasticsearch/reference/2.4/number.html>

**Comment by lukas-vlcek** [comment]
This PR is a bit dated. Shall we close it for now and reopen once it is needed?

**Comment by lukas-vlcek** [comment]
Cleanup, feel free to reopen if needed.

**Pull request closed without merging**